src/components/ShoppingListItem.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import * as timeframeConstants from '../lib/timeframeConstants';
+

```

Comment:
Are you using `timeFrameConstants` anymore?

Reply:
We plan on utilizing the constants throughout the app once fully implemented.

Reply:
Sounds good. If you're not using it in this file right now, though, you can remove line 2.